ingestion/setup.py

```diff
@@ -45,7 +45,9 @@ def get_long_description():
     "boto3~=1.19.12",
     "botocore==1.22.12",
     "avro-python3==1.10.2",
-    "grpcio-tools==1.48.2",
+    # https://github.com/grpc/grpc/issues/30843#issuecomment-1303816925
```

Comment:
taking the chance to update some versions to make our `grpcio` dependencies compatible with dagster base requirements